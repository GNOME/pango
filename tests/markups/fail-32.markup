<span insert-hyphens="blue">test</span>
