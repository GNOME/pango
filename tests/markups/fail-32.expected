ERROR: Value of 'insert_hyphens' attribute on <span> tag line 1 should have one of 'true/yes/t/y' or 'false/no/f/n': 'blue' is not valid